app/models/MovementMessage.scala

```diff
@@ -87,11 +90,12 @@ object MovementMessageWithStatus extends NodeSeqFormat with MongoDateTimeFormats
     (
       (__ \ "messageId").read[MessageId] and
         (__ \ "dateTime").read[LocalDateTime] and
+        (__ \ "received").read[LocalDateTime] and
```

Comment:
I've just realised there is a significant issue with this -- if a movement does not have a received date/time then the reads on this will fail. We're not going back to patch in the received time, so we _cannot_ merge this as is.